install_config/upgrading/manual_upgrades.adoc

```diff
@@ -908,6 +908,58 @@ in previous versions. There should be minimal performance impact to
 Elasticsearch while running this and it will not perform an install.
 ====
 
+[[manual-upgrading-cluster-metrics]]
+== Upgrading Cluster Metrics
+
+Use the following to upgrade an
+link:../../install_config/cluster_metrics.html[already-deployed Cluster Metrics install].
+
+To update to a newer version of the metrics components you will need to use the `refresh`
+mode of the deployer. This will stop all the metrics containers, update the metrics configuration
+files and redeploy the newer components. The `refresh` option will not change the metrics route or
+delete the metrics persistent volume claim.
+
+Metrics stored to persistent volumes will still contain the persisted data when the updated
+metrics are brought back online. 
+
+[IMPORTANT]
+====
+All non-persisted metric values will be lost during the metric `refresh`.
+
+All local changes to the metrics configurations will be overwritten with new configurations meant for the latest install.
+This includes things like number of instances in a replica set.
+====
+
+To perform the update you will need to follow the
+link:../../install_config/cluster_metrics.html#deploying-the-metrics-components[same steps]
+as when the metrics components were first deployed except this time you will need to specify the
+`MODE=refresh` option.
+
+You will also need to make sure you are using the correct template for your installation. Please see the following
+link:../../install_config/cluster_metrics.html#modifying-the-deployer-template[section] for the location of this template.
+
+For instance, if before you deployed metrics using:
+
+----
+$ oc new-app -f metrics-deployer.yaml \
+    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com
+----
+
+You will now need to run:
+----
+$ oc new-app -f metrics-deployer.yaml \
+    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,MODE=redeploy
```

Comment:
@mwringe 
Hmm, there is a discrepency between this example (which uses `redeploy`) and the introductory text (which uses `refresh`).  Which is correct?  (I will make the change in #2234.)